### ISSUE TITLE
Clarify style guide: what kinds of third party content do K8s docs allow?

### ISSUE THREAD

**Issue by zacharysarah** [kubernetes/website#15576]
## Improve the K8s style guide

The style guide needs to specify that K8s docs decline to host specific kinds of content:

1. Provider- or project-specific content from projects outside the kubernetes or kubernetes-sigs GitHub organizations

2. Identical content hosted in multiple locations ("dual-sourced" content)

For example, consider [Kubernetes in Docker](https://github.com/kubernetes-sigs/kind) (KinD):

Allowed | Action | Reason
---|---|---
✔️ | Linking to KinD docs from K8s docs | The KinD project resides in `github.com/kubernetes-sigs`.
❌ | Hosting KinD docs in k/website | Dual-sourced content requires duplicated effort from maintainers and tends to go stale/rotten more quickly.

## Why does this matter?

It's necessary to clarify which kinds of content we do and don't accept in K8s docs.

/sig docs
/priority important-soon

**Comment by zacharysarah** [comment]
@aimeeu This would be a great place for you to start.

**Comment by sftim** [comment]
Some example topics that could help illustrate the policy we settle on; what is and isn't OK.

- `cloud-controller-manager`
- Ingress
- load-balanced Services
- cluster auto scaling
- kube-proxy

**Comment by sftim** [comment]
The cluster API effort is also heavily tied to vendor-specific details.

**Comment by sftim** [comment]
Is it OK for pages that link to other CNCF projects (and separate organizations, ie not Kubernetes) to highlight the sibling relationship?

A few examples:
- containerd, rkt
- Prometheus, OpenMetrics, Thanos
- CoreDNS
- etcd
- Virtual Kubelet

**Comment by aimeeu** [comment]
@zacharysarah  While poking around the docs I noticed vendor-specific pages under _Tasks/Monitorin, Logging, and Debugging_  for Stackdriver and the Elastic stack. These pages also are targeted at Google Cloud Engine installations.  I didn't find an open issue to keep track of pages that contain vendor-specific content. Is there someplace we are logging these?

**Comment by zacharysarah** [comment]
@aimeeu 
> I didn't find an open issue to keep track of pages that contain vendor-specific content. Is there someplace we are logging these?

What would you recommend? Maybe opening an umbrella issue?

**Comment by zacharysarah** [comment]
@sftim 
> Is it OK for pages that link to other CNCF projects (and separate organizations, ie not Kubernetes) to highlight the sibling relationship?

Good point. Yes, I think that's OK--I'll update the style guide issue to reflect the change.

**Comment by aimeeu** [comment]
@zacharysarah  Yes, an umbrella issue sounds like a good approach. I will create one.
